### PR TITLE
[Wizard2] Constant var template part 1

### PR DESCRIPTION
**Pull request by beets**
Fill in the constant var template using old prediction code

- Displays detection results (missing drag/drop chip interface)
- Fixes small issues with csv header reading (last row & column index)
- Small CSS tweaks - added sections for separation

### CODE REVIEW

File: static/js/import_wizard2/components/mapping_sections/constant_var.tsx [hunk 1]
```diff
@@ -17,7 +17,57 @@
...
+          <tr>
+            <td>Date*:</td>
+            <td>[{dateMapping && dateMapping.column.header}]</td>
+          </tr>
+          <tr>
+            <td>Unit*:</td>
+            <td>[{unitMapping && unitMapping.column.header}]</td>
+          </tr>
+          <tr>
+            <td>Ignored columns:</td>
```
Comment (jehangiramjad) [inline static/js/import_wizard2/components/mapping_sections/constant_var.tsx:66]: nothing under Ignored columns?

**Review by jehangiramjad — APPROVED** [review]
Thanks!

File: static/js/import_wizard2/components/mapping_sections/constant_var.tsx (hunk 1 above)
Comment (beets) [reply static/js/import_wizard2/components/mapping_sections/constant_var.tsx:66]: Not yet. Added a TODO to fill it in.

**Comment by beets** [comment]
thanks for the quick review!

**Pull request merged**